compute/cker/include/cker/operation/Conv.h

```diff
@@ -119,19 +119,9 @@ class Conv
                        params.stride_height);
     }
 
-    if (_need_im2col)
-    {
-      uint8_t *im2col_raw_data = _im2col_data.data();
-      optimized::Conv(params, input_shape, input_data, filter_shape, filter_data, bias_shape,
-                      bias_data, output_shape, output_data, _im2col_shape, im2col_raw_data);
-    }
-    // TODO Apply optimized kernel if the optimized kernel is faster without im2col than the
-    // reference kernel
-    else
-    {
-      reference::Conv(params, input_shape, input_data, filter_shape, filter_data, bias_shape,
-                      bias_data, output_shape, output_data);
-    }
+    uint8_t *im2col_raw_data = _im2col_data.data();
+    optimized::Conv(params, input_shape, input_data, filter_shape, filter_data, bias_shape,
+                    bias_data, output_shape, output_data, _im2col_shape, im2col_raw_data);
```

Comment:
`im2col_raw_data` can be `nullptr`, but `optimized::Conv()` does not supports it as `nullptr`.

Reply:
If `_need_im2col` is false, `im2col_raw_data` is not used in `optimized::Conv()`. Tensorflow lite also uses this feature for optimization.

https://github.com/Samsung/ONE/blob/42a0337242798aaf8c5d8f2644f3ce58b848d19f/compute/cker/include/cker/operation/optimized/Conv.h#L107-L136

Reply:
I see, thanks for your explanation.